data/meson.build

```
@@ -0,0 +1,44 @@
+dbus_conf = configuration_data()
+
+dbus_conf.set(
+  'bindir',
+  join_paths(get_option('prefix'), get_option('bindir')),
+)
+
+configure_file(
+  input: 'org.gnome.Pomodoro.service.in',
+  output: 'org.gnome.Pomodoro.service',
```

You can you use to avoid the duplication:

```suggestion
  output: '@BASENAME@',
```